src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -312,6 +323,60 @@ class EthService(
     }
   }
 
+  def getBalance(req: GetBalanceRequest): ServiceResponse[GetBalanceResponse] = {
+    Future.successful {
+      withAccount(Address(req.address), req.block) { account =>
+        GetBalanceResponse(account.balance)
+      }
+    }
+  }
+
+  def getStorageAt(req: GetStorageAtRequest): ServiceResponse[GetStorageAtResponse] = {
+    import io.iohk.ethereum.rlp.UInt256RLPImplicits._
+
+    val byteArrayUInt256Serializer = new ByteArrayEncoder[UInt256] {
+      override def toBytes(input: UInt256): Array[Byte] = input.bytes.toArray[Byte]
+    }
+
+    val rlpUInt256Serializer = new ByteArraySerializable[UInt256] {
```

Comment:
Ideally MPT doesn't need to know about the objects it stores, but it should be easy enough to move those serializers elsewhere if we want to make the `mpt` package more uncoupled in the future.